File: DBBasic/output_obj/canacces_obj.cs
using DBBasic.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace DBBasic.output_obj
{
    public class canacces_obj : Serializable
    {
        public bool can;
        public class acces_permes : canacces_obj
        {
            public bool primera;
            public bool es_ilimitat;

            public acces_permes(bool primera, bool es_ilimitat)
            {
                this.can = true;
                this.primera = primera;
                this.es_ilimitat = es_ilimitat;
            }
        }
        public class acces_denegat : canacces_obj
        {
            public string motiu;

            public acces_denegat(string motiu)
            {
                this.can = false;
                this.motiu = motiu;
            }
        }
    }
}

File: Treballadors/app/src/main/java/com/app/treballadors/input_obj/canacces_obj.java
package com.app.treballadors.input_obj;

import com.app.treballadors.model.Serializable;

public class canacces_obj extends Serializable {
	public boolean can;
	public canacces_obj(boolean can) {
		this.can = can;
	}
	public class acces_permes extends canacces_obj{
		public boolean primera;
		public boolean es_ilimitat;

		public acces_permes(boolean primera, boolean es_ilimitat) {
			super(true);
			this.primera = primera;
			this.es_ilimitat = es_ilimitat;
		}
	}
	public class acces_denegat extends canacces_obj{
		public String motiu;

		public acces_denegat(String motiu) {
			super(false);
			this.motiu = motiu;
		}
	}
}
